### ISSUE TITLE
Slowness creeping up

### ISSUE THREAD

**Issue by lesfrerotsavelo** [gethugothemes/liva-hugo#54]
Hello

I started using this template a few months back and the performance is slowly getting worse over time.

Would you have any leads that would be worth looking into?

The website is lesfrerotsavelo.com.

Thank you,

**Comment by iwtga** [comment]
Yeah it's very slow, its unlikely its with the theme since Hugo is an SSG and the performance should be quite the opposite you are experiencing. It could probably be the Hosting Provider Issue.
